app/models/merchant.rb

```diff
@@ -1,6 +1,9 @@
 class Merchant <ApplicationRecord
   has_many :items, dependent: :destroy
   has_many :item_orders, through: :items
+  has_many :orders, through: :items
+  has_many :users, -> { where(role: 1)}
```

Comment:
Well done on figuring this out - I know this was hard won. Very impressed!

Reply:
Ditto!

Reply:
Nice! I would love to know more about how this is working